models/automations/view-automation-history.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: View an automation's history
+---
+
+<Note>
+This feature requires an Enterprise license. It is available only for:
+- W&B Multi-tenant Cloud
+- W&B Dedicated Cloud
+- W&B Self-Managed v0.75.0+
+</Note>
+
+This page describes how to view and understand the execution history of your W&B [automations](/models/automations), including what triggered an automation, what actions were taken, and whether they succeeded or failed.
+
+Each executed automation generates a record that includes:
+- **Execution timestamp**: When the automation was triggered.
+- **Triggering event**: The specific event triggered the automation.
+- **Status**: The execution's status. See [Execution status](#execution-status).
+- **Action details**: Information about what action was performed, such as notifying a Slack channel or running a webhook.
+- **Result details**: Additional information, if any, about the final outcome of executing the automation, including the error for a failed execution.
+
+Select a tab for details.
```

Comment:
Nit: Add a one liner describing what the tabs are for. Something like "Based on your use case, select the **Registry** or **Project** tab to learn how to view your automation's history".